apps/condo/domains/common/components/containers/GoogleTagManager.tsx

```
@@ -0,0 +1,33 @@
+    const { publicRuntimeConfig } = getConfig()
+    const { googleTagManagerId } = publicRuntimeConfig
+
+    return googleTagManagerId ?
+        <>
+            <script dangerouslySetInnerHTML={{
+                __html: `(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
+    new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
+    j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
+    'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
+  })(window,document,'script','dataLayer', "${googleTagManagerId}");`,
+            }}>
```

Biome (1.9.4)

[error] Avoid passing both children and the dangerouslySetInnerHTML prop.

This is the source of the children prop

Setting HTML content will inadvertently override any passed children in React

(lint/security/noDangerouslySetInnerHtmlWithChildren)